packages/epetra/cmake/Epetra_config.h.in

```diff
@@ -110,6 +110,9 @@
 /* Define if you have LAPACK library. */
 #cmakedefine HAVE_LAPACK
 
+/* Define if you have new LAPACK dgssvd3 and fgssvd3 functions which added a new parameter */
+#cmakedefine HAVE_LAPACK_GSSVD3
```

Comment:
I'm not keen on Epetra's highly generic macro names.  Would prefer prefixing in a way that's consistent with other Trilinos packages: `HAVE_EPETRA_LAPACK_GSSVD3`.